docs/source/api/apollo-gateway.mdx

```diff
@@ -143,7 +150,9 @@ You can specify any string value for the `name` field, which is used for identif
 
 ###### `introspectionHeaders`
 
-`Object | (service: ServiceEndpointDefinition) => Promise<Object> | Object`
+**This option is deprecated in favor of the drop-in replacement, [`IntrospectAndCompose`](#class-introspectandcompose).**
```

Comment:
Put this notice at the top of the description cell (as you did above) instead of at the top of the type cell